### PR TITLE
test: update egg-bin and adjust assert

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: boilerplate/_package.json [hunk 1]
```diff
@@ -3,30 +3,28 @@
...
   },
   "engines": {
     "node": ">=4.0.0"
   },
   "scripts": {
-    "test": "npm run lint && npm run test-local",
-    "test-local": "egg-bin test -r intelli-espower-loader",
-    "cov": "egg-bin cov -r intelli-espower-loader",
-    "lint": "eslint --ext js . --fix",
+    "test": "npm run lint -- --fix&& npm run test-local",
```
Comment (fengmk2) [inline boilerplate/_package.json]: 少了一个空格

File: package.json [hunk 2]
```diff
@@ -25,13 +25,11 @@
   "devDependencies": {
     "autod": "^2.7.1",
     "egg-ci": "^1.1.0",
-    "egg-init": "^1.5.0",
-    "eslint": "^3.10.2",
+    "egg-init": "^1.6.1",
+    "eslint": "^3.12.2",
     "eslint-config-egg": "^3.2.0",
-    "intelli-espower-loader": "^1.0.1",
     "npminstall": "^2.11.2",
```
Comment (fengmk2) [inline package.json:31]: 这个不需要依赖。。。

Comment (atian25) [reply package.json:31]: 需要的, 上面的 `install-deps` 用来单元测试的时候安装依赖

File: boilerplate/_package.json (hunk 1 above)
Comment (atian25) [reply boilerplate/_package.json]: 已改

**Review by popomore — APPROVED** [review]

**Pull request merged**